.pre-commit-config.yaml

```
@@ -9,7 +9,7 @@ exclude: |
 repos:
   - repo: local
     hooks:
-      - id: diff
+      - id: repo-structure-diff
```

```suggestion
      - id: diff
```
We can only apply this after another release